microk8s dbctl backup panics: etcdwrapper: key not found

When running `microk8s.dbctl backup` on a 3 node HA cluster (v1.20.7), I get the following error:

```
root@node13:~# microk8s.dbctl backup 
Backing up the datastore
INFO[0000] mode: backup-dqlite, endpoint: unix:///var/snap/microk8s/current/var/kubernetes/backend/kine.sock, dir: /tmp/tmpyfpmjp9c/backup-2021-06-15-10-25-47 
panic: etcdwrapper: key not found

goroutine 1 [running]:
main.check(...)
	/build/microk8s/parts/migrator/build/main.go:75
main.backup_dqlite(0xc36b28, 0x42, 0x7ffcdfc55396, 0x2b, 0x0, 0x0)
	/build/microk8s/parts/migrator/build/main.go:182 +0x887
main.run(0xc0000d0dc0)
	/build/microk8s/parts/migrator/build/main.go:283 +0x606
github.com/urfave/cli.HandleAction(0xb22fa0, 0xc441e8, 0xc0000d0dc0, 0xc0000d0dc0, 0x0)
	/build/microk8s/parts/migrator/go/pkg/mod/github.com/urfave/cli@v1.22.4/app.go:528 +0x58
github.com/urfave/cli.(*App).Run(0xc0000d6380, 0xc0000320a0, 0x5, 0x5, 0x0, 0x0)
	/build/microk8s/parts/migrator/go/pkg/mod/github.com/urfave/cli@v1.22.4/app.go:288 +0x655
main.main()
	/build/microk8s/parts/migrator/build/main.go:68 +0x4f6
Backup process failed. 2
```

Here is the inspection report of my cluster:

[inspection-report-20210615_102327.tar.gz](https://github.com/ubuntu/microk8s/files/6654038/inspection-report-20210615_102327.tar.gz)

Thank you

--- a/main.go
+++ b/main.go
@@ -192,16 +192,12 @@ func backup_dqlite(ep string, dir string) error {
 	}
 	for i, kv := range resp {
 		logrus.Debugf("%d) %s\n", i, kv.Key)
-		data, err := c.Get(ctx, string(kv.Key))
-		if err != nil {
-			return fmt.Errorf("couldn't get key %s: %w", kv.Key, err)
-		}
 
 		keyfile, datafile := getFileNamesForKey(i)
 		if err := os.WriteFile(keyfile, kv.Key, 0640); err != nil {
 			return fmt.Errorf("failed to write key file %q: %w", keyfile, err)
 		}
-		if err := os.WriteFile(datafile, data.Data, 0640); err != nil {
+		if err := os.WriteFile(datafile, kv.Data, 0640); err != nil {
 			return fmt.Errorf("failed to write data file %q: %w", datafile, err)
 		}
 	}

